src/client/tst/ClientTestFixture.h

```diff
@@ -379,7 +379,7 @@ class ClientTestBase : public ::testing::Test {
         mDeviceInfo.clientInfo.createStreamTimeout = TEST_DEFAULT_CREATE_STREAM_TIMEOUT;
         mDeviceInfo.clientInfo.stopStreamTimeout = TEST_DEFAULT_STOP_STREAM_TIMEOUT;
         mDeviceInfo.clientInfo.offlineBufferAvailabilityTimeout = TEST_DEFAULT_BUFFER_AVAILABILITY_TIMEOUT;
-        mDeviceInfo.clientInfo.loggerLogLevel = LOG_LEVEL_DEBUG;
+        mDeviceInfo.clientInfo.loggerLogLevel = LOG_LEVEL_SILENT;
```

Comment:
Can we have an env var driving this so we can change if we need verbosity on failed runs? This can be set manually

Reply:
👍 will ping you when updated.